File: DesktopSample/DataBinding/Section/LINQ/VB.NET/LINQ/ViewerForm.vb
Imports System.Text
Imports System.Xml
Imports GrapeCity.ActiveReports

Public Class ViewerForm

	Dim _manager As System.Resources.ResourceManager

	Sub New()
		Encoding.RegisterProvider(CodePagesEncodingProvider.Instance)
		InitializeComponent()
	End Sub

	'  LINQ to Object サンプル用の構造体定義
	Private Sub ViewerForm_Load(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.Load
		_manager = New System.Resources.ResourceManager(Me.GetType)

		' レポートを生成します。
		Dim rpt As New SectionReport
		rpt = New SectionReport
		rpt.LoadLayout(XmlReader.Create("..\\..\\..\\rptLINQtoObject.rpx"))

		arvMain.LoadDocument(rpt)
	End Sub

End Class

File: DesktopSample/DataBinding/Section/LINQ/C#/LINQ/ViewerForm.cs
using GrapeCity.ActiveReports;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using System.Xml;

namespace ActiveReports.Samples.LINQ
{
	public partial class ViewerForm : Form
	{
		private System.Resources.ResourceManager _manager;
		public ViewerForm()
		{
			_manager = new System.Resources.ResourceManager(typeof(ViewerForm));
			InitializeComponent();
		}
	  
		// LINQtoObject用の構造体を定義します。
		private void ViewerForm_Load(object sender, EventArgs e)
		{
			// レポートを生成します。
			var rpt = new SectionReport();
			rpt.LoadLayout(XmlReader.Create("..\\..\\..\\rptLINQtoObject.rpx"));
			// レポートを実行します。
			arvMain.LoadDocument(rpt);
		}
	}
}
